scripts/render.py

```diff
@@ -255,8 +258,9 @@ class RenderTrajectory:
     """Path to config YAML file."""
     rendered_output_names: List[str] = field(default_factory=lambda: ["rgb"])
     """Name of the renderer outputs to use. rgb, depth, etc. concatenates them along y axis"""
-    traj: Literal["spiral", "filename"] = "spiral"
-    """Trajectory to render."""
+    traj: Literal["spiral", "filename", "loop"] = "spiral"
```

Comment:
I was thinking a little more about the naming. Perhaps another term like `interpolate` would be more descriptive than `loop`

Reply:
Ah I see! Sorry for the confusion. :)

In this commit I've added this here https://github.com/ponimatkin/nerfstudio/commit/b8f7eb8a17ea35abec1c422db109d1eb3f088e2e. Then here https://github.com/ponimatkin/nerfstudio/commit/1df0ea87ab9fee0bd5570174e01ecd1a6b0e4c61 I've actually went further on and removed any mentions of the loops.